### ISSUE TITLE
您好，为什么第一个尺寸的模型可以跑通，其他两个尺寸的模型跑不通呀？错误如下：“InvalidArgumentError (see above for traceback): logits and labels must be broadcastable: logits_size=[128,2] labels_size=[32,2] [[node softmax_cross_entropy_with_logits_sg (defined at H:/自己的软件/代码文献/1/luna16_multi_size_3dcnn-master/new_model.py:267)。” 万分感谢，谢谢。

### ISSUE THREAD

**Issue by qishuaizheng** [shartoo/luna16_multi_size_3dcnn#16]

**Comment by shartoo** [comment]
请问你是训练过程有问题还是测试过程有问题？其他两个尺寸在训练和测试的时候需要传递的参数 使用的cube size不一样的，不知道你修改没？
